app/services/loader.js

```diff
@@ -13,7 +13,7 @@ export default Service.extend({
     if (config.isExternal) {
       options.url = url;
     } else {
-      options.url = config.withoutPrefix ? `${adapter.host}/${url}` : `${adapter.urlPrefix()}${url}`;
+      options.url = config.withoutPrefix ? `${adapter.host}${url}` : `${adapter.urlPrefix()}${url}`;
```

Comment:
@niranjan94 I tried doing that in first place, but consider a case in which we pass ```withoutPrefix: false``` but having leading url. In that case, it will remove the leading url and then the above code will be executed i.e ````${adapter.urlPrefix()}${url}``` which will make the url wrong as is happening in this case for the post request that we do to get the 'task_url'.